sql/service/src/main/java/org/apache/spark/sql/service/auth/SparkAuthFactory.java

```diff
@@ -146,7 +146,7 @@ public HiveAuthFactory(HiveConf conf) throws TTransportException, IOException {
         }
 
         // start delegation token manager
-        delegationTokenManager = new HiveDelegationTokenManager();
+        delegationTokenManager = new SparkDelegationTokenManager();
```

Comment:
Could we replace it with `org.apache.spark.deploy.security.HadoopDelegationTokenManager`?